### ISSUE TITLE
uniqueItems in object leads to TypeError

### ISSUE THREAD

**Issue by ypnos** [hyperjump-io/json-schema-validator#3]
On https://json-schema.hyperjump.io/ enter the schema:
```js
{
  "$schema": "https://json-schema.org/draft/2019-09/schema#",
  "type": "object",
  "properties": {
    "test": {
      "type": "object",
      "uniqueItems": true
   }
  }
}
```
The schema is declared **Valid** although `uniqueItems` is only expected for arrays. This is in line with other validators. Now enter a simple instance:
```js
{
  "test": {
  }
}
```
This results in **TypeError: se(...).map is not a function**

The problem would also occur on top-level. I used object "test" to illustrate that it only occurs when there is an instance of the object with `uniqueItems` attribute.

**Comment by jdesrosiers** [comment]
Thanks. I'm sure this will be an easy fix. I should have a fix published today.

**Comment by jdesrosiers** [comment]
Fixed and published as v0.7.1. Thanks again for the report!
